repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -2737,6 +2723,53 @@ public void repairAccesscontrolAlias(String guid) throws AtlasBaseException {
 
         RequestContext.get().endMetricRecord(metric);
     }
+
+    @Override
+    public void linkBusinessPolicy(String guid, List<String> linkGuids) throws AtlasBaseException {
+        processBusinessPolicy(guid, linkGuids, true);
+    }
+
+    @Override
+    public void unlinkBusinessPolicy(String guid, List<String> unlinkGuids) throws AtlasBaseException {
+        processBusinessPolicy(guid, unlinkGuids, false);
+    }
+
+    private void processBusinessPolicy(String guid, List<String> guids, boolean isLink) throws AtlasBaseException {
+        // Start recording the performance metrics for the operation
```

Comment:
We can divide function into two function, its okay to have two function for two different purpose, but merging linking and unliking in one place doesn't seem intuitive at all